### PR TITLE
Update due to SickRage conflict

### PR DESCRIPTION
**Pull request by xDowwdeen**
Have updated the file as per the edit i did myself to correct the installtion of SickRage.

Corrected the Repo
Corrected the unZip of the master.zip download

### CODE REVIEW

**Comment by MichaIng** [comment]
@xDowwdeen
Thanks for your PR.

However, please always open pull requests against our dev branch: https://github.com/Fourdee/DietPi/blob/dev/CONTRIBUTING.md

You changed the Sickrage Git repo to SickChill :thinking:. We already fixed it by changing to "SiCKRAGE", same repo but different capitalization. Hmm will check out what "SickChill" now is :smiley:.

**Comment by xDowwdeen** [comment]
@MichaIng Yes SickChill is from one of the original developers. There has always been "beef" between the two developers. (A lot to explain) 

The repo you were using originally was to the new SickChill repo. the developer of SiCKRAGE has manage to rediect the repo url you were using to his. Which is why you probs found why you had to reprogram parts of it all. 

Hope this sorta makes sense

**Comment by MichaIng** [comment]
Okay, it's just another renaming: https://github.com/SickChill/SickChill/commit/e92ab82a9fe8106132170c3a7113905788a08530

Hui first capitalization, now the whole name for changed. Thanks to bring this to our attention. Luckily we didn't yet release DietPi v6.17.

**Comment by MichaIng** [comment]
@Fourdee 
Ah okay, so jep now there are two repos/forks:
- https://github.com/SiCKRAGE/SiCKRAGE
- https://github.com/SickChill/SickChill

So it seems `SickChill` works with pre-v6.16 install method,
while for `SiCKRAGE` needed the adjustments we did with v6.17 so far: https://github.com/Fourdee/DietPi/commit/6203b3c908d75497680bee11291dad0344d4abf7, https://github.com/Fourdee/DietPi/commit/4bf058adc4a63ddb37ab297a4b94702e9d2d549b, https://github.com/Fourdee/DietPi/commit/c5260361fe65462e770f94e2c2db9d4db768e9e7
Although we have another report, that this does not work (any more: https://github.com/Fourdee/DietPi/issues/2126#issuecomment-429902381), so it seems `SiCKRAGE` is under heavy development due to new install method, requirements etc, while `SickChill` keeps it the old way.

SickChill install instructions: https://github.com/SickChill/SickChill/wiki/SickChill-installation-Ubuntu-15
- Looks indeed the same we did, besides we skipped install via git clone, but just download the archive directly.
SiCKRAGE install instructions: https://github.com/SiCKRAGE/SiCKRAGE#dependencies
- `pip install -r requirements.txt/sickrage`, python required as well.

Any suggestions why we should either go with the one or the other? Or support both? 🙈

**Comment by Fourdee** [comment]
@xDowwdeen 

> Yes SickChill is from one of the original developers. There has always been "beef" between the two developers. (A lot to explain)

Beef, chicken, fish or a carrot, this is not our concern.

What is our concern is offering a valid and working installation for our users. As long as "SiCKRAGE" legally exists on github.com, regardless of the current owner, we will offer it to our users and support it.
Should legality physically change for any software title, that removes the current project from existence, or, is at detriment to our users, we will remove the software title and support from our library.

Until then, I believe this PR request is best served as a software request for SickChill:
https://github.com/SickChill/SickChill
Please create a new ticket with a software request for this item.

In other words:
- Regardless of who claims what, "SiCKRAGE" is still an open source project, hosted on github.com, that we will offer and support for our users.
- Sonarr does not include childish non-sense attached to its project.
- C# is faster than Python
- Optimised code, runs more efficient than non-optimised code (regardless of language, even cross language)
- Always, facts (legally proven if need be) are more important than fiction (here-say).

**Comment by MichaIng** [comment]
@Fourdee 
Jep, agree. Only concern is that `SickChill` is actually closer to what `SickRage` was, at least about the install method. `SiCKRAGE` might be further away from what we supported before.
By staying with the "old" repo, we might just support the "winner" of the repo "first", instead of the perhaps more reasonable solution. But yeah, not our business and we have not much chance to do a qualified decision about which dev or fork now is "better" to support...

Btw. did you check: https://github.com/Fourdee/DietPi/issues/2126#issuecomment-429902381
- Perhaps something more changed about pre-reqs the last days?
